docs/rl-algorithms/rpo.md

```diff
@@ -0,0 +1,344 @@
+        | HalfCheetah-v4            | 2637.19 ± 1068.49                                            | 2716.51 ± 1314.93                                        |
+        | Hopper-v4                 | 2741.42 ± 269.11                                             | 2334.22 ± 441.89                                         |
+        | InvertedDoublePendulum-v4 | 5626.22 ± 289.23                                             | 5409.03 ± 318.68                                         |
+        | Reacher-v4                | -4.65 ± 0.96                                                 | -3.93 ± 0.19                                             |
+        | Swimmer-v4                | 124.88 ± 22.24                                               | 129.97 ± 12.02                                           |
+        | Pusher-v4                 | -30.35 ± 6.43                                                | -31.48 ± 9.83                                            |
+
+        Learning curves:
+        ![](../rpo/mujoco_v4_part2.png)
+
+        Tracked experiments:     
+
+        <iframe loading="lazy" src="https://wandb.ai/openrlbenchmark/cleanrl/reports/RPO-on-Mujoco_v4-Part-2--VmlldzozMjU3OTM4" style="width:100%; height:500px" title="RPO-mujoco-v4-part2"></iframe>
+        
+
+        Results with `rpo_alpha=0.5` (not tuned) on the tuned environments:
+
+        |                           | ppo_continuous_action_8M ({'tag': ['v1.0.0-13-gcbd83f6']})   | rpo_continuous_action ({'tag': ['pr-331']})   |
+        |:--------------------------|:-------------------------------------------------------------|:----------------------------------------------|
+        | Ant-v4                    | 1774.42 ± 819.08                                             | -7.99 ± 2.47                                  |
+        | HalfCheetah-v4            | 2667.34 ± 1109.99                                            | 2163.57 ± 790.16                              |
+        | Hopper-v4                 | 2761.77 ± 286.88                                             | 1557.18 ± 206.74                              |
+        | InvertedDoublePendulum-v4 | 5644.00 ± 353.46                                             | 296.97 ± 15.95                                |
+        | Reacher-v4                | -4.67 ± 0.88                                                 | -66.35 ± 0.66                                 |
+        | Swimmer-v4                | 124.52 ± 22.10                                               | 117.82 ± 10.07                                |
+        | Pusher-v4                 | -30.62 ± 6.80                                                | -276.32 ± 26.99                               |
+
+        Learning curves:
+        ![](../rpo/mujoco_v4_part2_0_5.png)
+
+
+        <!-- Tracked experiments: <iframe loading="lazy" src="https://wandb.ai/openrlbenchmark/cleanrl/reports/RPO-alpha-0-5-on-Mujoco_v4-Part-2--VmlldzozMjU4MTM1" style="height:500px;width:100%"> -->
```

Comment:
Is this duplicate info since we have the "Failure case of `rpo_alpha=0.5`:" down below?

Reply:
Oh nvm I see what you did there